### PR TITLE
Added plugin Mattacloud Beta

### PR DESCRIPTION
**Pull request by dougbrion**
Adding first release of Mattacloud Beta (v0.1.0) to the OctoPrint Plugin Repository.

Many cups of tea and biscuits have been consumed in producing this initial beta.
The following features have been implemented:

- Profile creation with account management
- Printer authentication workflow 
- The creation and deletion of printers
- Remote file management and file upload
- Live temperature data stream
- Control through GUI of temperatures, axis movement, extruder movement, flow rate, feed rate and more
- Storage and viewing of print history for each printer
- Capture of images during prints with live updates
- WebRTC video stream of Python 3 OctoPrint users

### CODE REVIEW

File: _plugins/mattacloud.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+Learn more about **Mattacloud** and its features - [https://mattalabs.com/products/mattacloud/](https://mattalabs.com/products/mattacloud/)
+
+_Note: **Mattacloud** is currently still in development so users may experience changes and errors during use._
+
+### Printer Setup
+
+The setup progress takes less than 5 minutes and consists of 5 simple steps. If you follow these, all the benefits of **Mattacloud** will apply to your printer. Happy printing!
+
+1. [Sign up](https://cloud.mattalabs.com/accounts/signup/) to the **Mattacloud** free beta trial. If you already have, just [login](https://cloud.mattalabs.com/accounts/login/). (Trial lasts for the duration of beta testing.)
```
Comment (jneilliii) [inline _plugins/mattacloud.md:68]: I think you should mention that once the public beta has expired there is no free tier subscription service here, best I can tell.

**Review by jneilliii — CHANGES_REQUESTED** [review]
@dougbrion Please see comment. Once that's complete I'll feel comfortable in merging.

File: _plugins/mattacloud.md (hunk 1 above)
Comment (dougbrion) [reply _plugins/mattacloud.md:68]: Will update with better description and info, the beta trial will convert into a free tier subscription service so will make this more clear!

**Comment by cp2004** [comment]
> WebRTC video stream of Python 3 OctoPrint users
Shameless plug by me: Be sure to link to [OctoPrint Upgrade To Python 3](https://github.com/cp2004/Octoprint-Upgrade-To-Py3) when someone asks why they can't get WebRTC streaming. Because believe me, they won't read. It won't be as bad as the wifi status plugin, since this *works* on Python 2, just not everything. Someone will still ask...

**Review by jneilliii — APPROVED** [review]

**Pull request merged**